src/grocker/builders.py

```diff
@@ -35,6 +35,7 @@ def build_root_image(docker_client, config, tag=None):
             os.path.join(build_dir, 'Dockerfile.j2'),
             os.path.join(build_dir, 'Dockerfile'),
             {
+                'image': config['system']['image'],
```

Comment:
Default is defined here: https://github.com/Polyconseil/grocker/blob/1ac2e17a5dbb93150b77188e176f838a69341a4f/src/grocker/resources/grocker.yaml#L9